var/spack/repos/builtin/packages/ppopen-appl-bem/package.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+
+class PpopenApplBem(MakefilePackage):
+    """
+    ppOpen-APPL/BEM is software used to support a boundary element analysis
+    executed on a parallel computer. The current version includes a software
+    framework for a parallel BEM analysis and an H-matrix library.
+    If you want to use the framework based on dense matrix computations,
+    please move to the directory 'src/framework' and
+    'src/framework_with_template'.
+    If you want to use the H-matrix library, please
+    move to the directly 'src/HACApK_with_BEM-BB-framework_1.0.0'.
+    """
```

Comment:
Just a minor comment:
```suggestion
    """ppOpen-APPL/BEM is software used to support a boundary element analysis
    executed on a parallel computer. 
    
    The current version includes a software
    framework for a parallel BEM analysis and an H-matrix library.
    If you want to use the framework based on dense matrix computations,
    please move to the directory 'src/framework' and
    'src/framework_with_template'.
    If you want to use the H-matrix library, please
    move to the directly 'src/HACApK_with_BEM-BB-framework_1.0.0'.
    """
```
I'm suggesting this change since when module files are generated the first paragraph is used for the short description and the whole docstring for the longer help.